app/views/members/_search.html.erb

```diff
@@ -0,0 +1,4 @@
+<%= form_tag(members_path, method: :get) do %>
+  <%= text_field_tag :query, params[:query] %>
```

Comment:
Thanks @phlipper I saw this on your original comment.  I could not find any references to sanitize in a form so assumed the model sanitize was enough.  Then I tried to get the syntax to work without success.  Finally after @elskwid's presentation tried what I just thought Rails might like.  No syntax error!